src/Core/src/Platform/Android/WindowExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Android.App;
+using Microsoft.Maui.Graphics;
+using AndroidX.Core.View;
+
+namespace Microsoft.Maui
+{
+	public static class WindowExtensions
+	{
+		public static void SetStatusBarColor(this Activity activity, Color color) =>
+			SetStatusBarColor(activity, color, color?.GetLuminosity() < 0.5);
+
+		public static void SetStatusBarColor(this Activity activity, Color color, bool lightForeground)
```

Comment:
We definitely want to get this added in since we already had this API in Forms

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs#L95

And doing it via extension like this is way better!!